src/routes/edit/delete-fragment-modal.tsx

```diff
@@ -14,6 +14,9 @@ export const DeleteFragmentModal = ({ id }: any) => {
 	const fragment = fragments.find((fragment: any) => fragment.id === id);
 
 	const deleteFragment = () => {
+		const undoHideFragment = () => {
```

Comment:
Am I seeing it right that this function is now inside deleteFragment and it was previously outside?

Reply:
Yes, somehow I managed to do that 😮

Reply:
This function is being called in only one place & the body of the function was a single function call - so I made it inline:

``` ts
const deleteFragment = () => {
// ...
		actionFunction: () => toggleFragmentVisibility(id, false),
// ...
};
```